tests/vertexonly/test_interpolation_from_parent.py

```diff
@@ -88,7 +96,7 @@ def pseudo_random_coords(size):
 # interpolation from a CG1 VectorFunctionSpace (I think)
 def test_scalar_spatialcoordinate_interpolation(parentmesh, vertexcoords):
     vm = VertexOnlyMesh(parentmesh, vertexcoords)
-    vertexcoords = vm.coordinates.dat.data_ro
+    vertexcoords = vm.coordinates.dat.data_ro.reshape(-1, parentmesh.geometric_dimension())
```

Comment:
What's the purpose of this change? Accounting for immersed manifolds?

Reply:
PyOP2 collapses the shape of a dat with shape `(whatever, 1)` into `(whatever,)`, so the broadcasting doesn't work.

Reply:
Can you add a comment saying as much to the first occurrence of this in each test file please, noting that it's for the interval case?